.github/workflows/setup/ubuntu_ompi.sh

```diff
@@ -19,7 +19,7 @@ sudo apt-get install -y --no-install-recommends \
     libfftw3-dev        \
     libopenmpi-dev      \
     openmpi-bin         \
-    libhdf5-openmpi-dev \
+    libhdf5-dev \
     python3             \
```

Comment:
Just nicely aligned newlines :D
```suggestion
    libhdf5-dev         \
    python3             \
```